doc/en/user/source/community/cog/index.rst

```diff
@@ -125,19 +125,15 @@ On the table below, replace the "$ALIAS$" template with HTTP or HTTPS or S3 if y
      - Default password (secret access key) for AWS basic authentication credentials
 
 Client configuration (System Properties)
------------------------------------------------------------------
-Note that all the settings reported in the previous tables can also be specified using System Properties instead of Environment variables.
+----------------------------------------
+Note that all the IIO  settings reported in the previous tables can also be specified using System Properties instead of Environment variables.
```

Comment:
IIO?

Reply:
Yep. there are several IIO_ prefixed settings in the previous tables.

![IIO_Settings](https://user-images.githubusercontent.com/1278116/99247504-51f7ab80-2807-11eb-8f87-1c61343ac2d8.jpg)